inventory/group_vars/all.yml

```diff
@@ -397,6 +397,9 @@ nginx_sites:
         proxy_set_header X-Forwarded-Ssl on;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
```

Comment:
I tested these three settings on a staging server and the `Host` header made a difference. We don't need the other two to solve the issue. Could you modify your PR to only add the one host header line?

I'm cautious about the forwarded-for header because of the security issue you referenced. We would need to add more to our configuration there. And maybe we won't need it.

Reply:
thanks for testing @mkllnk 
have removed the other two headers